crates/matrix-sdk-crypto/src/machine.rs

```diff
@@ -731,13 +730,22 @@ impl OlmMachine {
     ///
     /// **Note**: This does not need to be called for encrypted events since
     /// those will get passed to the `OlmMachine` during decryption.
+    #[deprecated(note = "Use OlmMachine::receive_verification_event instead", since = "0.7")]
     pub async fn receive_unencrypted_verification_event(
         &self,
         event: &AnyMessageLikeEvent,
     ) -> StoreResult<()> {
         self.verification_machine.receive_any_event(event).await
     }
 
+    /// Receive an verification event.
+    ///
+    /// This method can be used to pass verification events that are happening
+    /// in rooms to the `OlmMachine`.
```

Comment:
Maybe clarify that it should be a decrypted event (or clear event)? 
What about the `m.relates_to` field? it's not encrypted, should it be copy back in the resulting decrypted content?

Reply:
Hmm, would you know the type of the event if it was encrypted? Presumably no, but yeah, we can spell that out.